test/lib/i18n_test.rb

```diff
@@ -55,6 +55,20 @@ class I18nTest < ActiveSupport::TestCase
     end
   end
 
+  def test_en_for_raw_html
+    en = YAML.load_file(Rails.root.join("config/locales/en.yml"))
+    assert_nothing_raised do
+      check_values_for_raw_html(en)
+    end
+  end
+
+  def test_en_for_nil_values
```

Comment:
What is this actually checking for? How would a `nil` arise?

Reply:
I had a `nil` value in the branch I was working on when developing this PR, due to some fat fingering in a different commit. I had left a key with nothing after the colon while refactoring.

The `nil` value didn't make it past any pull review (I fixed it before submitting that other PR), so it's slightly hypothetical. But it did cause the other test to error out, so I thought it might be worth adding separately, so that you get a clean error message.

I'm happy to remove it if you think it's unnecessary or potentially confusing.